assets/styles/cloud.css

```diff
@@ -1,7 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
-
+@import url('https://fonts.googleapis.com/css2?family=Orbitron&display=swap');
```

Comment:
We already have Open Sans google font in `templates/layouts/layout.phtml` maybe we should move that one here also to be consistent ?

Reply:
Good Idea, I'll make that change now